index.js

```diff
@@ -96,6 +96,20 @@ if (debugMode) {
     console.log("DEBUG_MODE is active. To protect privacy, do not use in production.")
 }
 
+let ipv6_lcl = process.env.IPV6_LOCALIZE;
+if (ipv6_lcl === "false") {
```

Comment:
One last thing: this should also check for undefined as that results in the same behavior.
Could you simply do `!ipv6lcl && ipv6lcl === „false“`? `IPV6_LOCALIZE=0`should result in „0“ which is not falsy before it isn’t parsed as int, or is it?

And do we actually have to check for „false“? Isn’t the way to go to set the env var to ““ like so: `IPV6_LOCALIZE= npm start` or to undefined via `unset IPV6_LOCALIZE` which should both be falsy?

Reply:
Not sure if setting to 0 should be allowed, because it would mean the client IP would be localized to the empty string `''` so every client would be considered "local". Maybe that is actually useful? Not sure.

In any case `"0"` should not be considered the same as `false` just as `"1"` should not be considered `true`.

You're right we don't really need to handle/check for "false" as unset/undefined should have the same effect.

The only valid inputs for `IPV6_LOCALIZE` are `undefined`, or 1-8 (come to think of it, 1-7 actually- setting it to 8 would result in the same behavior as undefined)

So maybe the best thing is to accept `undefined || ''`, or an integer `1-7` specifically, and throw an error about any other values?

Reply:
> So maybe the best thing is to accept `undefined || ''`, or an integer `1-7` specifically, and throw an error about any other values?

I’d like that! I guess checking for`!ipv6lcl` should be enough then.